open-build-service-api/tests/integration.rs

```diff
@@ -47,11 +47,12 @@ fn create_authenticated_client(mock: ObsMock) -> Client {
 #[tokio::test]
 async fn test_package_list() {
     let mock = start_mock().await;
+    mock.add_project(test_project());
 
     let mtime = SystemTime::UNIX_EPOCH + Duration::new(10, 0);
     let srcmd5 = random_md5();
     mock.add_package_revision(
-        test_project(),
+        &test_project(),
```

Comment:
fwiw i find this very confusing ;  would be nicer to have a constant for the project name rather then a function call returning new copies of an owned string all the time

Reply:
Makes sense, originally this was quite a bit shorter since almost all the calls took `String`s, but I ended up gradually changing them to `&str`, so now there's no advantage.

Would it be fine to put that in a subsequent commit, instead of rebasing all of these on top of that change?

Reply:
Sure